app/src/main/java/github/daneren2005/dsub/fragments/SettingsFragment.java

```diff
@@ -109,6 +110,14 @@
 	public void onCreate(Bundle bundle) {
 		super.onCreate(bundle);
 
+		if(Build.VERSION.SDK_INT >= 21) {
+			CheckBoxPreference mediaButtons = (CheckBoxPreference) findPreference("mediaButtons");
```

Comment:
The `(CheckBoxPreference)` casting on the right side of variable assignments seems to be advised against in the current android-linter. It prefers just

```suggestion
 			CheckBoxPreference mediaButtons = findPreference("mediaButtons"); 
```